scripts/bump.js

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+
+const { bold, underline } = require('chalk');
+const { spawnSync } = require('child_process');
+const inquirer = require('inquirer');
+const semver = require('semver');
+
+const REPO_DIR = path.dirname(__dirname);
+
+const readJson = path => {
+  return JSON.parse(fs.readFileSync(path, { encoding: 'utf8' }));
+};
+
+const writeJson = (path, obj) => {
+  const content = JSON.stringify(obj, null, 2) + '\n';
+  fs.writeFileSync(path, content, { encoding: 'utf8' });
+};
+
+const PACKAGE_ORIG_VERSIONS = [
+  'cli',
+  'core',
+  'schema',
+  'legacy-scripting-runner'
+].reduce((result, packageName) => {
+  const packageJsonPath = path.join(
+    REPO_DIR,
+    'packages',
+    packageName,
+    'package.json'
+  );
+  const packageJson = readJson(packageJsonPath);
+  result[packageName] = packageJson.version;
+  return result;
+}, {});
+
+const ensureMainPackageVersionsAreSame = () => {
+  if (
+    !(
+      PACKAGE_ORIG_VERSIONS.cli === PACKAGE_ORIG_VERSIONS.core &&
```

Comment:
it would be great to break this assumption eventually (I still can't think of a reason we need them to match up as long as all versions in play are a semi-ish-recent one), but this is fine here for now.

Reply:
Yeah, we'll need to revisit here when we want to make an effort to decouple cli and core versions.